tests/integration/targets/iosxr_ospfv2/vars/main.yaml

```diff
@@ -0,0 +1,356 @@
+            default_cost: 6
+        cost: 2
+        dead_interval: 2
+        default_metric: 10
+        demand_circuit: 'enable'
+        external_out: 'disable'
+        flood_reduction: 'disable'
+        hello_interval: 1
+        mtu_ignore: 'enable'
+        packet_size: 577
+        passive: 'disable'
+        priority: 1
+        process_id: '30'
+        retransmit_interval: 2
+        router_id: '2.2.2.2'
+        summary_in: 'enable'
+        transmit_delay: 2
+        weight: 2
+
+overridden:
+  commands:
+    - router ospf 10
+    - no authentication keychain ansible_test1102
+    - no area 11 default-cost 5
+    - no area 11 cost 11
+    - no area 22 default-cost 6
+    - router ospf 30
+    - no cost 2
+    - no weight 2
+    - no passive disable
+    - no priority 1
+    - no flood-reduction disable
+    - no default-metric 10
+    - no router-id 2.2.2.2
+    - no demand-circuit enable
+    - no packet-size 577
+    - no transmit-delay 2
+    - no summary-in enable
+    - no external-out disable
+    - no dead-interval 2
+    - no hello-interval 1
+    - no retransmit-interval 2
+    - no mtu-ignore enable
+    - no area 11 default-cost 5
+    - no area 22 default-cost 6
+    - router ospf 27
+    - area 20 authentication keychain ansi11393
+    - area 20 default-cost 2
+    - area 20 cost 2
+  after:
+    processes:
+      - process_id: '10'
+      - adjacency_stagger:
+          max_adjacency: 20
+          min_adjacency: 10
+        authentication:
+          message_digest:
+            keychain: 'ansible1101pass'
+        process_id: '26'
+      - areas:
+          - area_id: '10'
+            authentication:
+              keychain: 'ansi11393'
+            hello_interval: 2
+          - area_id: '20'
+            authentication:
+              keychain: 'ansi11393'
+            cost: 2
+            default_cost: 2
+        process_id: '27'
+      - process_id: '30'
+deleted:
+  commands:
+    - router ospf 10
+    - no authentication keychain ansible_test1102
+    - no area 11 default-cost 5
+    - no area 11 cost 11
+    - no area 22 default-cost 6
+    - router ospf 26
+    - no adjacency stagger 10 20
+    - no authentication message-digest keychain ansible1101pass
+    - router ospf 27
+    - no area 10 authentication keychain ansi11393
+    - no area 10 hello-interval 2
+    - router ospf 30
+    - no cost 2
+    - no weight 2
+    - no passive disable
+    - no priority 1
+    - no flood-reduction disable
+    - no default-metric 10
+    - no router-id 2.2.2.2
+    - no demand-circuit enable
+    - no packet-size 577
+    - no transmit-delay 2
+    - no summary-in enable
+    - no external-out disable
+    - no dead-interval 2
+    - no hello-interval 1
+    - no retransmit-interval 2
+    - no mtu-ignore enable
+    - no area 11 default-cost 5
+    - no area 22 default-cost 6
+
+  after:
+    processes:
+      - process_id: '10'
+      - process_id: '26'
+      - process_id: '27'
+      - process_id: '30'
```

Comment:
All these process ids are mentioned in the deleted.yml testcase. If I am right , I think all the processes should be deleted by 'no router ospf' command .

Reply:
same as above

Reply:
Process entries should be deleted entirely if we're not having a separate module just for ospf vrf.

Reply:
We have decided to have separate rm for vrf support.